pkg/certs/cert-inspection/certgraphanalysis/collector.go

```diff
@@ -145,7 +160,7 @@ func gatherFilteredCerts(ctx context.Context, kubeClient kubernetes.Interface, a
 	}
 
 	pkiList := PKIListFromParts(ctx, inClusterResourceData, certs, caBundles)
-	return pkiList, errors.NewAggregate(errs)
+	return pkiList, apierrors.NewAggregate(errs)
```

Comment:
`utilerrors` by convention.  `apierrors` is for the one in apimachinery with `IsNotFoundError`